src/hooks/api/useGames.ts

```diff
@@ -78,11 +94,40 @@ export function useGames() {
     return createdParticipation
   }
 
+  const createOrJoinNextGame = async () => {
```

Comment:
Could we add a tag based on the game name or game ID, to determine if a game that was started stems from a already played game?

game1
"prevGame" : "false" 

--> game was closed and repoened via "nextGame"

game1
"prevgame" = game1

[Like this it would be vulnerable to games having the same name but would generally allow to introduce a filtered list to the game summary screen so that everyone in that game could join, as well as allowing us to filter our list in the Lobby such that games started from closed games don't show up and have to go "public" to be restarted]

Reply:
I would say it's new feature that the games created with "Start new Game" are protected against other people joining it.

We can add this as a new story, but i don't think we will have time to implement id.

If we had time, i would rather add a flag visibility to the game, that you can make a game public (visible in the lobby) or private (only accessible via invite link or "Start new Game")